project/SbtMultiJvm.scala

```diff
@@ -167,10 +168,11 @@ object MultiJvmPlugin extends AutoPlugin {
       // the first class wins just like a classpath
       // just concatenate conflicting text files
       assembly / assemblyMergeStrategy := {
-        case n if n.endsWith(".class") => MergeStrategy.first
-        case n if n.endsWith(".txt")   => MergeStrategy.concat
-        case n if n.endsWith("NOTICE") => MergeStrategy.concat
-        case n                         => (assembly / assemblyMergeStrategy).value.apply(n)
+        case n if n.endsWith(".class")  => MergeStrategy.first
+        case n if n.endsWith(".txt")    => MergeStrategy.concat
+        case n if n.endsWith("NOTICE")  => MergeStrategy.concat
+        case n if n.endsWith("LICENSE") => MergeStrategy.concat
```

Comment:
Although this change was needed for this feature specifically, I think it would be wiser to make this change in a separate PR because in general we should concatenate all `LICENSE` statements.

Reply:
Should I seperate this to a dedicated PR? but seems like only Netty 4 contains this thing.

Reply:
Yes please make a separate PR for this. Although it's true that this only effects netty 4 for now, in general of there multiple `LICENSE`'s we do want to concat them (i.e. this isn't just specific to netty 4)